packages/RNTester/js/examples/Button/ButtonExample.js

```diff
@@ -26,92 +25,131 @@ exports.description = 'Simple React Native button component.';
 
 exports.examples = [
   {
-    title: 'Simple Button',
-    description: ('The title and onPress handler are required. It is ' +
-      'recommended to set accessibilityLabel to help make your app usable by ' +
-      'everyone.': string),
+    title: 'Button with default styling',
+    description: '',
```

Comment:
You don't need to provide an empty string to the description, it's an optional field.